vignettes/explanation-limitations-and-known-issues.Rmd

```diff
@@ -7,11 +7,39 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-A Rhino app must live in git root for GitHub Actions CI to work.
+# GitHub Actions CI
```

Comment:
I don't like this explanation. I don't think it actually explains a specific topic.... And unnecessarily sounds bad.

Instead, i suggest:
GitHub Actions CI --> How to: Rhino app outside repository root
Box --> Explanation: Using box modules

Reply:
Good point, I split the article.